EIPS/eip-233.md

```diff
@@ -18,13 +18,89 @@ Today discussions about hard forks happen at various forums and sometimes in ad-
 
 ## Specification
 
-A Meta EIP should be created and merged as a *Draft* as soon as a new hard fork is planned. This EIP should contain:
+A Meta EIP should be created and merged as a *Draft* as soon as a new hard fork is planned. The hard fork should be 6 - 9 months after the last hard fork.
+
+This EIP should contain:
 - the desired codename of the hard fork,
-- list of all the EIPs included in the hard fork and
-- activation block number once decided and
+- activation block number once decided
+- a timeline section
+- an EIPs to include section
 - the **Requires** header should point to the previous hard fork meta EIP.
 
-The draft shall be updated with summaries of the decisions around the hard fork. It should move in to the `Accepted` state once the changes are frozen (i.e. all referenced EIPs are in the `Accepted` state) and in to the `Final` state once the hard fork has been activated.
+The draft shall be updated with summaries of the decisions around the hard fork.
+
+### Timeline
+
+Once a timeline with key dates is agreed upon for other crucial dates. The basic outline of a hardfork timeline should include:
+* Hard deadline to accept proposals for this hard fork
+* Soft deadline for major client implementations
+* Projected date for testnet network upgrade
+* Projected date for mainnet upgrade (the activation block number / projected date for this block)
+
+### EIP Inclusion Process
+
+Anyone that wishes to propose a Core EIP for the hard fork should make a PR against the Meta EIP representing the hard fork. The EIP must be published as at least `Draft`. It enters the _Proposed EIPs_ section, along with at least one person who is a point of contact for wanting to include the EIP.
+
+Once the EIP has been accepted by Core Devs, the EIP should be moved to the _Accepted EIPs_ section. If the EIP has major client implementations and no security issues by the timeline date, it is scheduled for inclusion.
```

Comment:
`it is scheduled for inclusion` -> Does this mean that it is moved from the _Accepted EIPs_ section to the _Included EIPs_ one? 

`the timeline date` -> Does this mean the _Soft deadline for major client implementations_ date?